changelogs/fragments/1320-Zos_mvs_raw_ignores_tmp_hlq.yml

```diff
@@ -0,0 +1,5 @@
+bugfixes:
+  - zos_mvs_raw - mvs raw ignored the tmphlq argument by the creation of dd's.
+    Fix now enables the user to pass tmphlq argument to be use by the creation of
```

Comment:
Module option is listed as `tmp_hlq` which I think is the spelling we should stick to. 
I differ with the whole changelog fragment, since the user does not specifically pass the tmp_hlq to any dd names through the interface how would they care about an inner function change? 

I propose we change the wording.